desktop/apps/categories2/index.coffee

```diff
@@ -1,6 +1,6 @@
 express = require 'express'
 routes = require './routes'
-adminOnly = require '../../lib/middleware/admin_only'
+adminOnly = require '../../lib/admin_only'
```

Comment:
`admin_only` was the only middleware that was used not globally in `desktop`, so I decided to move all of the middleware up to ./lib/middleware and get rid of the folder.